### PR TITLE
Akka 2.6.20

### PR DESCRIPTION
**Pull request by patriknw**

### CODE REVIEW

**Review by ennru — APPROVED** [review]
LGTM.

File: blog/_posts/2022-09-06-akka-2.6.20-released.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+* Alpakka 4.0.0
+* Alpakka Kafka 3.0.1
+* Akka Persistence JDBC 5.1.0
+* Akka Persistence Cassandra 1.0.6
+* Akka Projections 1.2.5
+
+## Credits
+
+Special thanks to [@He-Pin](https://github.com/He-Pin) for all additions and improvements to Akka Streams.
+Also cheers to [@Captain1653](https://github.com/Captain1653) for his tierless effort to add API links to the Akka documentation.
```
Comment (franciscolopezsancho) [inline blog/_posts/2022-09-06-akka-2.6.20-released.md:55]: tireless?

File: blog/_posts/2022-09-06-akka-2.6.20-released.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+short: We are pleased to announce Akka 2.6.20 and releases of other Akka projects
+category: news
+tags: [releases]
+---
+
+Dear hakkers,
+
+We are pleased to announce a new patch release of Akka 2.6. Noteworthy additions and improvements in this release:
+
+* Convient API for `mergeAll`, `concatAll`, `interleaveAll` and `alsoToAll` in Akka Streams, thanks to [@He-Pin](https://github.com/He-Pin), [#19045](https://github.com/akka/akka/issues/19045)
```
Comment (johanandren) [inline blog/_posts/2022-09-06-akka-2.6.20-released.md:14]:
```suggestion
* Convenient API for `mergeAll`, `concatAll`, `interleaveAll` and `alsoToAll` in Akka Streams, thanks to [@He-Pin](https://github.com/He-Pin), [#19045](https://github.com/akka/akka/issues/19045)
```

File: blog/_posts/2022-09-06-akka-2.6.20-released.md (hunk 1 above)
Comment (patriknw) [reply blog/_posts/2022-09-06-akka-2.6.20-released.md]: thanks, fixed

**Review by johanandren — APPROVED** [review]

**Pull request merged**